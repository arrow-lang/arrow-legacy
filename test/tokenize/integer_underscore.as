// Integer
12_2

// Identifier
_231

// Integer <dot> Identifier
34._21

// Float
432.32_1
