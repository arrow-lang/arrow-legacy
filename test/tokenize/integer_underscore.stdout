test/tokenize/integer_underscore.as:2.1-5: integer: 122
test/tokenize/integer_underscore.as:5.1-5: identifier: _231
test/tokenize/integer_underscore.as:8.1-3: integer: 34
test/tokenize/integer_underscore.as:8.3-4: punctuator: .
test/tokenize/integer_underscore.as:8.4-7: identifier: _21
test/tokenize/integer_underscore.as:11.1-9: float: 432.321
test/tokenize/integer_underscore.as:12.1-2: end
